### ISSUE TITLE
[Mobile APP] Add build support

### ISSUE THREAD

**Issue by Pentoos** [inventree/inventree-app#287]
It would be great to add the build support to the mobile app.

**Comment by SchrodingersGat** [comment]
How would you like to see this implemented? How would you make use of this?

**Comment by Pentoos** [comment]
The build feature excellent.
It would be usefull to be able to use it from a phone or a tablet.
For example : 
     - validate outputs 
     - validate a test /add attachments like pictures directly from the phone
     - add notes etc...

**Comment by SchrodingersGat** [comment]
@Pentoos makes sense, that would be useful. Also, potentially a reasonable amount of work to implement.

If you want to see this developed soon, you could potentially [sponsor the feature](https://github.com/sponsors/inventree) - otherwise it will be added to the (long) list of wishlist features :)

**Comment by reilly-callaway** [comment]
Stock allocation for build orders using barcodes would be nice to see here too.

Workflow would look something like:
 * Open stock allocation page/section for build order
 * Press allocate stock from barcodes "button"
 * Scan a barcode
     * Only accepts stock that is in the BOM/variants/substitues
 * Shouldn't require any selecting of which part you're allocating the stock for
 * Should always return to the barcode scanner screen after allocating a stock ready for the next item.

**Comment by NerosKi** [comment]
The way @reilly-callaway describes the workflow is also exactly what would be useful for us. Currently this is a lot of manual input lookup up the batch/lot numbers of the actual stock that goes into the build order.

**Comment by cbenhagen** [comment]
I am starting to work on this..
